pipeline/luigi/CompileVCFFiles.py

```diff
@@ -1245,6 +1246,148 @@ def run(self):
 
         copy(self.output().path, PipelineParams().output_dir)
 
+
```

Comment:
Added this to make sure all sources follow the same pattern -- i.e. they all get copied into the output directory.